highfidelity/hifi
# Login dialog rework

Based on true story in: https://highfidelity.fogbugz.com/f/cases/7895/HMD-Tablet-login-refactoring

Test plan:
- Try login/logout/create account in Tablet/Desktop modes
- Try getting to login dialog from Snapshot app in both Tablet/Desktop modes
- check if all controls in place and looks ok
- check virtual keyboard in HMD, ensure it's in the correct place and isn't visually clipping through any other control
